src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/shared-module/components/base.component.ts

```diff
@@ -26,6 +28,10 @@ export abstract class BaseComponent implements OnDestroy {
     this.subscription.add(value);
   }
 
+  pictureParkUIConfig$ = new BehaviorSubject(
```

Comment:
- Check the permissions in the config factory
- load the config in an APP_INITIALIZER